launcher/agent/agent.go

```diff
@@ -98,7 +98,7 @@ func (a *agent) Close() error {
 func (a *agent) MeasureEvent(event cel.Content) error {
 	a.tpmMu.Lock()
 	defer a.tpmMu.Unlock()
-	return a.cosCel.AppendEvent(a.tpm, cel.CosEventPCR, defaultCELHashAlgo, event)
+	return a.cosCel.AppendEventPCR(a.tpm, cel.CosEventPCR, defaultCELHashAlgo, event)
```

Comment:
If you use the AppendEvent func, all of the changes can be server or cel

Reply:
change back to AppendEvent